.github/workflows/ci.yml

```
@@ -22,6 +22,29 @@ jobs:
       - name: Run Markdown Lint
         uses: actionshub/markdownlint@master
 
+  ecr:
+    runs-on: kubernetes
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: eu-west-1
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1.2.2
```

```suggestion
        uses: aws-actions/amazon-ecr-login@v1
```

Might be worth pinning to v1 rather than the patch versions of the Action?

---

yeah makes sense, I think this was a desperate attempt to get it to work